COVIDSafepassage/COVIDSafepassage/permission.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import permissions
+
+from passsystem import views
+from passsystem.models import User
+
+
+class IsAuthenticated(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        # allow login and logout requests without authentication
+        if isinstance(view, views.SessionLoginApiView) or isinstance(view, views.SessionLogoutApiView)\
```

Comment:
why logout?

Reply:
We may still want to invalidate the cookie even if the session is unauthorized. Can we always assume the session has to be authenticated for logout?
For instance, what if the cookie if already timedout or a scenario of logout after expiry time.
Or what if the firebase revoked the cookies once the user got removed from firebase.

So the cookie may still remain on the android client and we may still want to invalidate the cookies without authentication